bigbluebutton-web/grails-app/conf/bigbluebutton.properties

```diff
@@ -377,7 +375,7 @@ presentationBaseURL=${bigbluebutton.web.serverURL}/bigbluebutton/presentation
 beans.presentationService.presentationBaseUrl=${presentationBaseURL}
 #----------------------------------------------------
 # Inject values into grails service beans
-beans.presentationService.presentationDir=${presentationDir}
+#presentationDir=${presentationDir}
```

Comment:
Prefer removing it instead of comment! If will not be necessary anymore.

Reply:
Set to be as it used to